src/components/timeline/TrackContextMenu.js

```diff
@@ -100,14 +108,35 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
     }
   };
 
-  _isolateTrack = () => {
-    const { isolateProcess, isolateLocalTrack, rightClickedTrack } = this.props;
+  _isolateProcess = () => {
```

Comment:
I recommend going up to `Diff settings` in the GitHub UI and changing it to split view, it makes these changes much easier to read.